integration/graphql-types.ts

```diff
@@ -29,6 +29,15 @@ export type HasNameTypes = AuthorId | Book;
 
 export type FieldWithArgsTypes = AuthorId | Book;
 
+export type UnionResolvers = {
+  UnionProp: { __resolveType(o: String | Boolean): string };
+  SearchResult: { __resolveType(o: AuthorId | Book): string };
```

Comment:
😅 Maybe Authors and Books should be best-left for Onboarding Examples? Not actually implemented in our codebase? 😬

Reply:
Oh nvm I thought this was BluePrint or GraphQL or something. Never seen this repo.

Reply:
Ah yeah, this is just a toy schema for the integration tests of the code generator.